client/src/components/common/AppBar/style.css

```diff
@@ -0,0 +1,11 @@
+.root{
+  position: fixed;
+}
+.toolbar{
+  background-color: white;
+  justify-content: space-between;
+}
+.logo{
```

Comment:
@alidah0 It might be a good idea to make these class names a little more specific, so you don't have a problem later if another component has a `.logo` or `.root` and accidentally gets these styles applied to it.  Something like `.app-bar` instead of `.root`, and then you could do `.app-bar .logo`.